package.json

```diff
@@ -16,7 +16,7 @@
     "type-check:cypress": "tsc --project cypress/tsconfig.json",
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx --max-warnings 0 --report-unused-disable-directives",
     "lint:fix": "pnpm lint -- --fix",
-    "prettier-base": "prettier . --ignore-path .gitignore --ignore-unknown",
+    "prettier-base": "prettier . --ignore-unknown",
```

Comment:
@HonkingGoose This change was also needed. We were using a custom `ignore-path` so `.prettierignore` wasn't considered for excluding files or folders.